pkg/subscriptions/subscriptions-view.jsx

```diff
@@ -0,0 +1,247 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define([
+    "base1/cockpit",
+    "subscriptions/react",
+    "subscriptions/cockpit-components-listing",
+], function(cockpit, React, cockpitListing) {
+
+"use strict";
+
+var _ = cockpit.gettext;
```

Comment:
> Since this is a new file I would suggest consistent indentation with subscriptions-register.jsx. One has 4 spaces in front of each line, this one doesn't.

fixed